### DISCUSSION TITLE
Styrbar OTA 33816598 - should I update?

### DISCUSSION THREAD

**Discussion by theincogtion** [Koenkk/zigbee2mqtt#23668] [Devices]
![image](https://github.com/user-attachments/assets/6703fbd1-462b-40f6-b586-2c39670a54fb)
Should I update or should I wait?

**Comment by JackTalisker** [comment]
I updated to 2.4.16 (33816598) few days ago and my styrbars work well as expected.

**Comment by b2un0** [comment]
i update three of them and i got this issues.
https://github.com/Sian-Lee-SA/Home-Assistant-Switch-Manager/issues/248

so the hold up and hold down actions are now hard to use.

**Reply by theincogtion** [reply]
> i update three of them and i got this issues. [Sian-Lee-SA/Home-Assistant-Switch-Manager#248](https://github.com/Sian-Lee-SA/Home-Assistant-Switch-Manager/issues/248)
> 
> so the hold up and hold down actions are now hard to use.

Is the issue now solved? Save to update?

**Reply by b2un0** [reply]
this is no issue, this is the new way they are working.
se my last comments in the linked issue
